src/components/Menu.vue

```diff
@@ -86,6 +93,16 @@ export default defineComponent({
       userProfile: 'user/getUserProfile'
     })
   },
+  methods: {
+    setEComStore(event: CustomEvent) {
+      if(this.userProfile) {
+        this.store.dispatch('user/setEcomStore', {
+          'eComStore': this.userProfile.stores.find((store: any) => store.productStoreId == event['detail'].value)
+        })
+      }
+      emitter.emit("selectedShop")
```

Comment:
I think we can name it something like `changeProductStore` in place of `selectedShop`

Reply:
OK, Sir, I will correct it.